azure-arc-for-servers/get-kvsecrets-from-arc-servers.ps1

```diff
@@ -0,0 +1,65 @@
+# run the following on your Azure ARC machines to get secrets from you Key Vault
+function Get-AccessToken {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [string] $audience,
+        [string] $apiVersion = "2019-08-15"
+    )
+
+    $azcmagent = azcmagent show
```

Comment:
It looks like this script is dependent on an external executable. You need to document the prerequisites. Be sure to include required version of PowerShell.

You may also want to test for prerequisites or add error handling.

Reply:
good point. Added. thx

Reply:
Where does the user get `azcmagent`? That is not explained.

Reply:
It's in the readme